sql/bduni_convert.sql

```diff
@@ -13,29 +13,33 @@ CREATE USER MAPPING FOR {user}
 
 GRANT USAGE ON FOREIGN SERVER bduni_server TO {user};
 
-DROP FOREIGN TABLE IF EXISTS troncon_de_route CASCADE;
-DROP FOREIGN TABLE IF EXISTS non_communication CASCADE;
-IMPORT FOREIGN SCHEMA public LIMIT TO (troncon_de_route, non_communication)
+DROP FOREIGN TABLE IF EXISTS {output_schema}.troncon_de_route CASCADE;
+DROP FOREIGN TABLE IF EXISTS {output_schema}.non_communication CASCADE;
+
+IMPORT FOREIGN SCHEMA {input_schema} LIMIT TO (troncon_de_route, non_communication)
 FROM SERVER bduni_server
 INTO public;
```

Comment:
The entire file seems ok. one question for this line : I think this line should be "INTO {output_schema};", no ?

Reply:
@lgrd  well done ! I didn't catch this one. I will update the .sql file.